docs/tethys_sdk/gizmos.rst

```diff
@@ -128,39 +116,49 @@ Inserts a Gizmo at the location of the tag.
 
 *Parameters*:
 
-* **name** (string or literal) - The name of the Gizmo to insert as either a string (e.g.: "date_picker") or a literal (e.g.: date_picker).
 * **options** (dict) - The configuration options for the Gizmo. The options are Gizmo specific. See the Gizmo Showcase documentation for descriptions of the options that are available.
 
-*Examples*:
+*Example*:
 
 ::
 
-    # With literal for name parameter
-    {% gizmo date_picker date_picker_options %}
-
-    # With string for name parameter
-    {% gizmo "date_picker" date_picker_options %}
+    {% gizmo date_picker_options %}
 
 **register_gizmo_dependency**
```

Comment:
I still don't love calling this "register_gizmo_dependency". How about "import_gizmo_dependency"? Should it be plural? "import_gizmo_dependencies"?